### PR TITLE
Update to the aptos-node-checker binary

### PR DESCRIPTION
**Pull request by sirouk**
### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3230)

### CODE REVIEW

**Comment by banool** [comment]
Thanks Sirouk! Will take a look soon.

File: developer-docs-site/docs/nodes/node-health-checker.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ Depending on your setup, you may want to check out a particular branch, to ensur
 Run NHC:
 
 ```
-cargo run --release -- server run --baseline-node-config-paths /etc/nhc/ait2_validator.yaml /etc/nhc/devnet_fullnode.yaml
+cargo run --bin aptos-node-checker server run --baseline-node-config-paths /etc/nhc/ait3_validator.yaml /etc/nhc/devnet_fullnode.yaml
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/node-health-checker.md:174]: Also, alternatively -p aptos-node-checker would work as well.

File: developer-docs-site/docs/nodes/node-health-checker.md [hunk 2]
```diff
@@ -121,7 +121,7 @@ Next, download these configuration YAML files into the `/etc/nhc` folder in your
 ```
 mkdir /etc/nhc
 cd /etc/nhc
-configs=(single_node_validator devnet_fullnode ait2_validator); for c in ${configs[@]}; do wget https://raw.githubusercontent.com/aptos-labs/aptos-core/main/ecosystem/node-checker/configurations/$c.yaml; done
+configs=(single_node_validator devnet_fullnode ait3_validator); for c in ${configs[@]}; do sudo wget https://raw.githubusercontent.com/aptos-labs/aptos-core/main/ecosystem/node-checker/configuration_examples/$c.yaml; done
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/node-health-checker.md:124]: This is a good catch!  Most probably do not have permissions to `/etc/nhc`.  Though, the permissions will likely need to be changed to allow for it.

Let's instead do a `sudo chown $USER /etc/nhc` prior to this command, which will then allow the user to have access to these files without running `sudo` later.

**Review by gregnazario — COMMENTED** [review]
@sirouk appreciate the help!  Just a comment on how we should probably not use sudo multiple times.

File: developer-docs-site/docs/nodes/node-health-checker.md (hunk 2 above)
Comment (sirouk) [reply developer-docs-site/docs/nodes/node-health-checker.md:124]: Great point. I would guess that changing ownership of the file after and not elevating the wget would be a better practice from a security standpoint.

**Comment by davidiw** [comment]
closing an oldr pr.

**Pull request closed without merging**